File: benchmark/pi.py
#!/usr/local/bin/python3

n = 10000000

pi = 0
for i in range(n) :
    if i % 2 == 0 :
        pi += (1 / (i * 2 + 1))
    else :
        pi -= (1 / (i * 2 + 1))

print(pi * 4)

File: benchmark/pi.lua
local n = 10000000

local pi = 0
for i = 0, n do
    if i % 2 == 0 then
        pi = pi + (1 / (i * 2 + 1))
    else
        pi = pi - (1 / (i * 2 + 1))
    end
end

print(pi * 4)
